dynamic_rest/serializers.py

```diff
@@ -290,6 +290,13 @@ def get_plural_name(cls):
             )
         return cls.Meta.plural_name
 
+    def get_request_method(self):
+        return getattr(
+            self.context.get('request'),
+            'method',
+            ''
+        )
```

Comment:
maybe call `.upper()` on this just in case the underlying value ever becomes lower-case